### PR TITLE
DSD-279: CSS Color Vars Malformed in Production Build

### PR DESCRIPTION
**Pull request by nypl-wluisi**
Fixes #DSD-279

## **This PR does the following:**
`@mixin css-vars` was generating the css color vars for white and black as hex values not strings:

```
:root {
  --ui-#000: #000;
  --ui-#fff: #fff;
```

The fix here wraps the colors "black" and "white" in quotes, to prevent them from being converted to hex color values.

### Front End Review:
- [ ] View [the example in Storybook](https://deploy-preview-563--stoic-murdock-c7f044.netlify.app/?path=/story/form-elements-button--basic)

### CODE REVIEW

**Review by bigfishdesign13 — APPROVED** [review]
LGTM

**Review by EdwinGuzman — APPROVED** [review]
LCA works with this PR branch so approving. There's still the spacing issue but it doesn't necessarily need to be a part of this PR.

File: src/styles/01-colors/_colors-utility.scss [hunk 1]
```diff
@@ -32,8 +32,8 @@ $warning-secondary: #ec7b1f;
 
 //Mapping where we associate a name string with each SASS var
 $colors: (
-  black: $black,
-  white: $white,
+  "black": $black,
```
Comment (hokei) [inline src/styles/01-colors/_colors-utility.scss:37]: Can you explain why the quotes are here, in a comment

Comment (nypl-wluisi) [reply src/styles/01-colors/_colors-utility.scss:37]: @hokei I added a comment explaining this, to hopefully prevent someone from removing the quotes in the future.

**Pull request merged**